README.md

```diff
@@ -3,6 +3,16 @@
 ## Project's Description 
 A React Native onboard project for Taqtile
 
-## Environment and tools * 
-## Steps to run and debug *
+## Environment and tools 
 
+- Node.js 16.15.1
+- Android Studio 4.2.1
+- Npm 8.5.5
+
+## Steps to run and debug
+
+After you start a virtual device on Android Studio run:
```

Comment:
Nos "steps to run and debug", seria legal incluir os comandos para instalação das dependências, que é necessário antes de rodar o projeto. A ideia do README é pensar: se uma pessoa que nunca trabalhou no projeto for contribuir, o README está claro para que ela consiga?